ipa-multipoint/src/main/java/org/hyperledger/besu/ethereum/trie/verkle/IPAHasher.java

```diff
@@ -0,0 +1,14 @@
+package org.hyperledger.besu.ethereum.trie.verkle;
+
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.bytes.Bytes32;
+import org.hyperledger.besu.nativelib.ipamultipoint.LibIpaMultipoint;
+
+
+public class IPAHasher implements Hasher<Bytes32> {
```

Comment:
```suggestion
public class PedersenCommitment implements Hasher<Bytes32> {
```
Can we rename to something like this? Since IPAHasher will be used in final "proof" IMO? IPA is built on top of pedersenCommitment, let's think about IPA when we finish the trie structure and commitments

Reply:
Ok, no problem. We just have to keep in mind that there is a naming clash for commit. We have commit for saving to the database, and commit as in computing a cryptographic commitment. Hence the choice of hash here. But you are right for IPA, not great...